packages/unding/src/cli.js

```diff
@@ -1,17 +1,38 @@
 #!/usr/bin/env node
 
 import { program } from "commander";
+import { join } from 'node:path';
+import * as vite from 'vite';
 
 import { startServer } from './server.js';
 
-async function start() {
-  await startServer({ cwd: process.cwd() });
+async function build() {
+  await Promise.all([
+    vite.build({
+      configFile: join(process.cwd(), 'vite.config.js'),
+    }),
+
+    vite.build({
+      configFile: join(process.cwd(), 'vite.config.js'),
+      build: {
+        ssr: true
+      }
+    })
+  ]);
```

Comment:
That can't be right - is there no way to run `vite.build` once and emit client and server entries at the same time?